source/_integrations/ai_task.markdown

```
@@ -0,0 +1,62 @@
+      for:
+        minutes: 10
+  actions:
+    - action: ai_task.generate_data
+      data:
+        task_name: "garage door left open comment"
+        instructions: "Generate a funny notification that garage door was left open"
+      response_variable: generated_text
+    - action: notify.mobile_app
+      data:
+        message: "{{ generated_text.data }}"
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)